### PR TITLE
Add release step

### LINKED ISSUE #8: Add Release Action to CI Workflow

### PR DESCRIPTION
**Pull request by s5dsn-eqee**
Close: #8

### CODE REVIEW

**Comment by s5dsn-eqee** [comment]
@alamb How do you feel about it? 
Do you think we’re ready to release version 0.1.0?

**Comment by alamb** [comment]
> @alamb How do you feel about it? Do you think we’re ready to release version 0.1.0?

I am not quite sure how this (seemingly magic) github action works, but I do think we should release verion 0.1.0 🚀

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -68,3 +68,31 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if all needed jobs succeeded
+        uses: re-actors/alls-green@release/v1
+        with:
+          jobs: ${{ toJSON(needs) }}
+
+  release:
+    needs: [check]
+    if: "success() && startsWith(github.ref, 'refs/tags/')"
```
Comment (s5dsn-eqee) [inline .github/workflows/ci.yml:84]: Ensure the release only happens on a successful check and when the ref is a tag.

**Review by austin362667 — APPROVED** [review]

**Comment by austin362667** [comment]
Thank you @dmitrybugakov 
IMO I think it's ok to do this, I've seen similar CI tricks in release check in https://github.com/pydantic/pydantic-core/blob/e3eff5cb8a6dae8914e3831b00c690d9dee4b740/.github/workflows/ci.yml#L709

**Pull request merged**